guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p> </p><ul><li>Adds Multiple-GPU training support to:</li></ul><p><code>        <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#maskrcnn" target="_blank">MaskRCNN</a></code></p><p><code>         <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#unetclassifier" target="_blank">Unet</a></code></p><p><code>         DeepLab</code></p><p><code>         PSPNetClassifier</code></p><p><code>        <s> </s><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#retinanet" target="_blank"><s>RetinaNet</s></a><s> </s>   #not added for this model</code></p><p> </p><p> </p><ul><li>Model Extension support is also added</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Removed that model from the list!</p>

Reply:
<p>Thanks for reviewing <a href="https://github.com/priyankatuteja" target="_blank">priyankatuteja</a>!</p>